space-concordia-robotics/robotics-prototype
# Update nav 138

# Assignee Section

## Description

We have finished issue #138. READ the description to know what it is about. **It is NOT about creating a path system. It's just about making a basic GUI for navigation. Please do not request changes out of scope of this issue**. You can make new issues though.

closes #138

## Testing

0. `cd` into `robot/rospackages`, then `catkin_make` and then `sb`
1. `startgui`, then `rosgui`, and `roslaunch local_rover.launch` in "~/Programming/robotics-prototype/robot/util/rosRoverStart".
2. Open `localhost:5000/navigation`, set antenna parameters (longitude: 45, lattitude: 45, bearing: 1) and add some markers.
3. Open page the same page on another browser or refresh with <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>R</kbd>.
4. Verify that the info persists.
5. Modify info on one page and verify that it changes on another without a refresh.

**Please read the description of the PR again**

Approval from at least one software team lead is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
